Include tests/__init__.py in sdist

itemadapter-0.9.0.tar.gz from PyPI ships `tests` but not `tests/__init__.py`, so running tests in it is impossible:

```
tests/test_utils.py:7: in <module>
    from tests import (
E   ModuleNotFoundError: No module named 'tests'
```

(not sure why it isn't packaged)

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,2 +1,5 @@
-include LICENSE
-include README.md
+include Changelog.md
+include pytest.ini
+include tox.ini
+
+recursive-include tests *.py *.txt

